autogpt/commands/file_operations_utils.py

```diff
@@ -146,7 +146,7 @@ def is_file_binary_fn(file_path: str):
 
 def read_textual_file(file_path: str, logger: logs.Logger) -> str:
     if not os.path.isfile(file_path):
-        raise FileNotFoundError(f"{file_path} not found!")
+        raise FileNotFoundError(f"read_file: {file_path}: no such file or directory")
```

Comment:
Nit: that's a lot of colons, doesn't help with readability

Reply:
I'm copying how the command line works:

```
$ cat sfasdf
cat: sfasdf: No such file or directory
```

Reply:
Or, to be clear: I agree with you. I would prefer only one colon. I guess that decision was made like 30 years ago, oh well.

Reply:
The example you give results in the message `Error: cat: sfasdf: No such file or directory`. Three colons is too much for me to parse at a glance.
```suggestion
        raise FileNotFoundError(f"read_file {file_path} failed: no such file or directory")
```